### PR TITLE
Stream api and lambdas task1

### PR DESCRIPTION
**Pull request by BulavaKatsyaryna**

### CODE REVIEW

**Review by ArtikTico — APPROVED** [review]
good job!

File: src/main/java/LECTURE_Stream_API_And_Lambdas/Task1/Demo.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package LECTURE_Stream_API_And_Lambdas.Task1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Demo {
+
```
Comment (KaydunovDenis) [inline src/main/java/LECTURE_Stream_API_And_Lambdas/Task1/Demo.java:8]: delete

File: src/main/java/LECTURE_Stream_API_And_Lambdas/Task1/Demo.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    public static void main(String[] args) {
+
+        Random randomGenerator = new Random();
```
Comment (KaydunovDenis) [inline src/main/java/LECTURE_Stream_API_And_Lambdas/Task1/Demo.java:11]: I thing what this variable can be called simply
randomGenerator - > random

File: src/main/java/LECTURE_Stream_API_And_Lambdas/Task1/Demo.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+        List<Integer> listOfRandomNumbers = new ArrayList<Integer>();
+        while (listOfRandomNumbers.size() < 20) {
+            int a = randomGenerator.nextInt(100);
+            if (!listOfRandomNumbers.contains(a)) {
+                listOfRandomNumbers.add(a);
+            }
```
Comment (KaydunovDenis) [inline src/main/java/LECTURE_Stream_API_And_Lambdas/Task1/Demo.java:18]: You can remove it and do replacing:
ArrayList -> Set

**Review by KaydunovDenis — COMMENTED** [review]
Like it! See comment.

**Pull request merged**